shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -172,6 +172,13 @@
     public static final String THREAD_POOL_ACTIVE_THREADS = PREFIX + "segmentstore.thread_pool.active_threads";  // Histogram
 
     // Metrics in Controller
+    // KeyValueTable request counts
+    public static final String CREATE_KVTABLE = PREFIX + "controller.kvtable.created";
+    // segment counts
+    public static final String KVTABLE_SEGMENTS_COUNT = PREFIX + "controller.kvtable.segments.count";
+    public static final String CREATE_KVTABLE_LATENCY = PREFIX + "controller.kvtable.created_latency_ms";
```

Comment:
Be consistent. `created` -> `create`

Reply:
This is as per the STREAM metric name : `public static final String CREATE_STREAM_LATENCY = PREFIX + "controller.stream.created_latency_ms";`